microsoft/PowerToys
# [Launcher] Fix PowerToys Plugin Setup

## Summary of the Pull Request

During the development of the plugin the setup has been reworked and I missed some pieces... Really really sorry! 😓 

Changed the plugin icon with a more generic one.
It would be nice to have a PowerToys icon in fluent style 😃 


## PR Checklist

- [x] **Closes:** #26152


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- Build setup
- Install
- Launcher PowerToys plugin should be installed and fully working
